docs/logs/customizing-the-sdk/README.md

```diff
@@ -2,15 +2,28 @@
 
 ## OpenTelemetryLoggerProvider
 
-TODO
+As shown in the [getting-started](../getting-started/README.md) doc,
+calling the extension method `ILoggerBuilder.AddOpenTelemetry` will register
+`OpenTelemetryLoggerProvider` as an `ILoggerProvider` using the default
+configuration. By itself, this is not particularly useful. The subsequent
+sections show how to build a more useful provider.
 
-## Building the OpenTelemetryLoggerProvider
+## OpenTelemetryLoggerProvider configuration
 
-TODO
+The following concepts can be viewed in the example app
+[Program.cs](Program.cs)
 
-## OpenTelemetryLoggerProvider configuration
+`OpenTelemetryLoggerProvider` holds the logging configuration
+`OpenTelemetryLoggerOptions`, which includes the following:
 
-TODO
+1. The list of
+   [Processors](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/logging-library-sdk.md#logprocessor),
+   including exporting processors which exports logs to
```

Comment:
"processor which exports logs to exporters" sounds weird.

I would suggest "including exporting processors which pass log records to exporters".

Reply:
I'm happy to include a diagram here, but I'm not sure what should be included.
- Should we map the data flow? (i.e. LoggerFactory > Logger > LoggerProvider > Exporter)
- or should we attempt to illustrate the configuration options?